app/services/threat_level/assessment_runs_data_builder_service.rb

```diff
@@ -0,0 +1,18 @@
+module ThreatLevel
+  class AssessmentRunsDataBuilderService
```

Comment:
Actually that is Clean code-sque way of implementing things.

https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882

If you insist on compressing code into less classes I can do it but don't want to :)